docs/release-notes.md

```diff
@@ -13,6 +13,7 @@ Major changes:
 Minor changes:
 
 - Detect truncated xz archives
+- iso extract minimal-iso: Fix corrupt output when writing to stdout
```

Comment:
Minor: WDYT about

```suggestion
- iso: Fix corrupt `extract minimal-iso` output when writing to stdout
```

for consistency?

(Came up while reviewing this PR with @prestist.)